src/index.js

```diff
@@ -1,5 +1,16 @@
 import './style.css';
-// eslint-disable-next-line no-unused-vars
-import Game from './modules/game.js';
+import { postScore, displayScores } from './modules/game.js';
 
-// const game = new Game();
+// Event listener to add a score
+document.querySelector('#add-score').addEventListener('click', (e) => {
+  e.preventDefault();
+  postScore();
```

Comment:
[OPTIONAL] I think it is a good idea to do a validation and show an error message to prevent the user to send blank entries. It will help to improve the user's experience. :eyes: :eyes: